src/Application/Console/Formatters/Console.php

```diff
@@ -0,0 +1,349 @@
+<?php
+
+declare(strict_types=1);
+
+namespace NunoMaduro\PhpInsights\Application\Console\Formatters;
+
+use NunoMaduro\PhpInsights\Application\Console\Contracts\Formatter;
+use NunoMaduro\PhpInsights\Application\Console\Style;
+use NunoMaduro\PhpInsights\Domain\Contracts\HasDetails;
+use NunoMaduro\PhpInsights\Domain\Insights\ForbiddenSecurityIssues;
+use NunoMaduro\PhpInsights\Domain\Insights\InsightCollection;
+use NunoMaduro\PhpInsights\Domain\Metrics\Architecture\Classes as ArchitectureClasses;
+use NunoMaduro\PhpInsights\Domain\Metrics\Architecture\Files;
+use NunoMaduro\PhpInsights\Domain\Metrics\Architecture\Globally as ArchitectureGlobally;
+use NunoMaduro\PhpInsights\Domain\Metrics\Architecture\Interfaces as ArchitectureInterfaces;
+use NunoMaduro\PhpInsights\Domain\Metrics\Architecture\Traits as ArchitectureTraits;
+use NunoMaduro\PhpInsights\Domain\Metrics\Code\Classes;
+use NunoMaduro\PhpInsights\Domain\Metrics\Code\Code;
+use NunoMaduro\PhpInsights\Domain\Metrics\Code\Comments;
+use NunoMaduro\PhpInsights\Domain\Metrics\Code\Functions;
+use NunoMaduro\PhpInsights\Domain\Metrics\Code\Globally;
+use NunoMaduro\PhpInsights\Domain\Metrics\Complexity\Complexity;
+use NunoMaduro\PhpInsights\Domain\Results;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class Console implements Formatter
+{
+    /** @var Style */
+    private $style;
+
+    public function __construct(InputInterface $input, OutputInterface $output)
+    {
+        $this->style = new Style($input, $output);
+    }
+
+    /**
+     * Outputs the summary according to the format.
+     *
+     * @param Results $results
+     * @param string  $dir
+     *
+     * @return self
+     */
+    public function summary(Results $results, string $dir): self
```

Comment:
This function (and the followings) can be private as only the `format` function use them ?